public/scripts/GithubAPIService.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Class for making request to Github API
+ */
+class GithubAPIService {
+
+		constructor() {
```

Comment:
constructor without params doesn't make sense here. 
Better to use constant API_URL imported from consts.js